### PR TITLE
feat: ProAltLogo

### PR DESCRIPTION
**Pull request by dreamwasp**
## Overview
This creates a new version of our Pro Alt Logo that allows dev to choose the inner text color. 

### PR Checklist

- [x] Related to designs: [LX-4320](https://codecademy.atlassian.net/browse/LX-4320)
- [x] Related to JIRA ticket: [LX-4320](https://codecademy.atlassian.net/browse/LX-4320)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### Description
Previously this effect had been achieved by loading in an image (as it is [here](https://www.codecademy.com/learn/paths/computer-science)). This logo also should improve the load time of page.

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.4.4-alpha.013613.0<br/>@codecademy/styleguide@17.7.6-alpha.013613.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.a7cb1e.0<br/>@codecademy/styleguide@17.8.1-alpha.a7cb1e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f919d83ca535f231cfa8c19--gamut-preview.netlify.app>https://5f919d83ca535f231cfa8c19--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f919d83ca535f231cfa8c19>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f919d8663b4ab24b5fefaab--gamut-preview.netlify.app>https://5f919d8663b4ab24b5fefaab--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f919d8663b4ab24b5fefaab>Deploy Logs</a></p>

File: packages/gamut-labs/src/brand/ProLogoAlt/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export type ProLogoAltProps = {
+  ariaLabel?: string;
+  backgroundColor?: string;
+  textColor?: string;
+  width?: number;
+};
+
+const backgroundDefault = colors.yellow;
+const textDefault = colors.black;
+const widthDefault = 50;
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/brand/ProLogoAlt/index.tsx:13]: Nit: is there any reason these are declared separately & only used once? IMO it'd be a little more readable to put these directly below:

```ts
  width = 50,
```

File: packages/gamut-labs/src/brand/ProLogoAlt/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+export const ProLogoAlt: React.FC<ProLogoAltProps> = ({
+  ariaLabel = 'ProLogo',
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/brand/ProLogoAlt/index.tsx:16]: Nit: I think this'd be a little more informative / easy to read for screenreaders:

```suggestion
  ariaLabel = 'Codecademy Pro Logo',
```

**Review by JoshuaKGoldberg — APPROVED** [review]
LGTM, thanks! Glad this work came to fruition :)

File: packages/styleguide/stories/Labs/Brand/Atoms/ProLogoAlt.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      control: { type: 'select', options: selectableColors },
+    },
+  }}
+/>
+
+# ProLogoAlt
+
+<StoryStatus status="volatile" />
+
+New version of the Pro Alt that allows you to chose the text color. Use sparingly, and make sure it has a lot of visual distinction to stand out appropriately.
```
Comment (JoshuaKGoldberg) [inline packages/styleguide/stories/Labs/Brand/Atoms/ProLogoAlt.stories.mdx:24]: Nit: "new" is a relative term, and will get outdated pretty quickly.

```suggestion
Version of the Pro Alt logo that allows you to chose the text color. Use sparingly, and make sure it has a lot of visual distinction to stand out appropriately.
```

File: packages/gamut-labs/src/brand/ProLogoAlt/index.tsx (hunk 1 above)
Comment (dreamwasp) [reply packages/gamut-labs/src/brand/ProLogoAlt/index.tsx:13]: some weirdness i haven't quite sussed out causes the object key to be converted into strings when declared inside the SVG function (like, the fill was being set to the string "colors.yellow" as the default) - i just added width to that same place for consistency. i will remove the width line + lmk if you might know why the string conversion thing is happening

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.ac939e.0<br/>@codecademy/styleguide@17.8.1-alpha.ac939e.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.bed5e9.0<br/>@codecademy/styleguide@17.8.1-alpha.bed5e9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f91a368441f3e2f05dc3c35--gamut-preview.netlify.app>https://5f91a368441f3e2f05dc3c35--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f91a368441f3e2f05dc3c35>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f91a388522cb22663ac18b8--gamut-preview.netlify.app>https://5f91a388522cb22663ac18b8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f91a388522cb22663ac18b8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f91a4c11daf4c2d187e9c6e--gamut-preview.netlify.app>https://5f91a4c11daf4c2d187e9c6e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f91a4c11daf4c2d187e9c6e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.eb5de3.0<br/>@codecademy/styleguide@17.8.1-alpha.eb5de3.0<br/></pre>

File: packages/gamut-labs/src/brand/ProLogoAlt/index.tsx (hunk 1 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/brand/ProLogoAlt/index.tsx]: Huh, weird. I haven't seen this before. Do you have a commit showing the code weirdness? I'll move to Slack :)

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/brand/ProLogoAlt/index.tsx]: Glorious. We're following up on Slack & #1072 ; I think it's a Storybook bug...

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.86f72a.0<br/>@codecademy/styleguide@17.8.1-alpha.86f72a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f91de52b20a6311031d58e2--gamut-preview.netlify.app>https://5f91de52b20a6311031d58e2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f91de52b20a6311031d58e2>Deploy Logs</a></p>

File: packages/styleguide/helpers/selectableColors.ts [hunk 4]
```diff
@@ -10,3 +10,12 @@ export const selectableColors = Object.keys(colors).reduce<
     [colorKey]: colorAtKey,
   };
 }, {});
+
+export const selectableColorNames = Object.keys(colors).reduce<
```
Comment (JoshuaKGoldberg) [inline packages/styleguide/helpers/selectableColors.ts:14]: @jakemhiller @codecaaron just calling your attention to this: it's beautiful. 💎 ✨

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.e34d65.0<br/>@codecademy/styleguide@17.8.1-alpha.e34d65.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f92f0574e10a600c6d3340d--gamut-preview.netlify.app>https://5f92f0574e10a600c6d3340d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f92f0574e10a600c6d3340d>Deploy Logs</a></p>

File: packages/styleguide/helpers/selectableColors.ts (hunk 4 above)
Comment (codecaaron) [reply packages/styleguide/helpers/selectableColors.ts:14]: :)

For some ⛳ we could just do `Object.keys(colors)` as storybook accepts an array for control options (and then converts it a map like this).

But this is great :D

**Review by codecaaron — APPROVED** [review]
Love it, wonderful :)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.5.1-alpha.c4cbbd.0<br/>@codecademy/styleguide@17.8.1-alpha.c4cbbd.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f93163198c2a510c0789e1e--gamut-preview.netlify.app>https://5f93163198c2a510c0789e1e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f93163198c2a510c0789e1e>Deploy Logs</a></p>

**Pull request merged**